src/handlers/autolabel.rs

```diff
@@ -97,16 +87,27 @@ pub(super) async fn parse_input(
 
                 // Treat the following situations as a "new PR":
```

Comment:
We should only apply the `trigger_files` labels when the PR was pushed to or when it was opened/converted from draft.